### PR TITLE
custom_image_host: add snapcraft aliases

### PR DESCRIPTION
**Pull request by sergiusens**
The aliases point to snapcraft-$base using images from cloud images.

Signed-off-by: Sergio Schvezov <sergiusens@gmail.com>

### CODE REVIEW

**Comment by townsend2010** [comment]
I don't think we want to use `snapcraft:core16` unless we are saying `snapcraft` is a new custom remote.  Is that the case here?  #387 doesn't list is like this.

**Comment by townsend2010** [comment]
Fixes #387

**Comment by Saviq** [comment]
> Is that the case here?

It just felt right... we have a way of "grouping" images/aliases so thought it could be the right thing to do?

Otherwise, which remote does "snapcraft-core16" belong to?

File: src/daemon/custom_image_host.cpp [hunk 1]
```diff
@@ -49,6 +49,56 @@ auto multipass_default_aliases(mp::URLDownloader* url_downloader)
 
     default_aliases.insert({"core", core_image_info});
 
+    // Image information on 16.04
+    const QString url_16{"https://cloud-images.ubuntu.com/releases/16.04/release/ubuntu-16.04-server-cloudimg-amd64-disk1.img"};
+    const auto last_modified = url_downloader_16->last_modified({url_16});
```
Comment (townsend2010) [inline src/daemon/custom_image_host.cpp]: `last_modified` has already been declared.  Either rename to be unique or fix the previous declaration to not be `const` and drop `auto` here.

Also in this line, `url_downloader_16` has not been declared anywhere.

File: src/daemon/custom_image_host.cpp [hunk 2]
```diff
@@ -49,6 +49,56 @@ auto multipass_default_aliases(mp::URLDownloader* url_downloader)
...
+        if (line.endsWith("ubuntu-16.04-server-cloudimg-amd64-disk1.img"))
+        {
+            hash = QString(line.split(' ').first());
+            break;
+        }
+    }
+
+    // Image information on 18.04
+    const QString url_18{"https://cloud-images.ubuntu.com/releases/18.04/release/ubuntu-18.04-server-cloudimg-amd64.img"};
+    const auto last_modified = url_downloader_18->last_modified({url_18});
```
Comment (townsend2010) [inline src/daemon/custom_image_host.cpp]: Same as above, `last_modified` has already been declared and `url_downloader_18` has not been declared:)

**Comment by townsend2010** [comment]
After a bit of discussion with @Saviq, it probably makes sense to use a special `snapcraft` remote for this so it would be in the form of `snapcraft:core16`, etc.  If this indeed the intent, then I will take over this and add support for the special `snapcraft` remote:)

**Comment by townsend2010** [comment]
I've proposed #398 in place of this PR.

**Pull request closed without merging**